docker-traceability-api/pom.xml

```diff
@@ -132,6 +73,20 @@
       <groupId>com.github.docker-java</groupId>
       <artifactId>docker-java</artifactId>
       <version>${docker-java.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpclient</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpmime</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.glassfish.hk2</groupId>
+          <artifactId>hk2-utils</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
This is a tricky one. This dependency, by using the `maven-dependency-plugin` [includes the sources](https://github.com/hk2-project/hk2/blob/master/hk2-utils/pom.xml#L100) of `org.jvnet.tiger-types`. On the other hand, the core brings its own dependency of `org.jvnet.tiger-types` (transitive dependency of stapler)

From baseline 2.2 where stapler version is bumped to [1.243](https://github.com/jenkinsci/jenkins/blob/53f665c4539c270b8745481cc9380c28b58c8a1b/core/pom.xml#L42) (and transitively `tiger-types` to 2.2), the PCT fails since stapler calls a method that is not present in the version of tiger types that `hk2-utils` includes in its source code, so this dependency needs to be excluded so that the right version of `tiger-types` is used.